lib/util/http/host-rules.ts

```diff
@@ -19,6 +19,8 @@ export function applyHostRules(url: string, inOptions: any): any {
   } else if (token) {
     logger.trace('Applying Bearer authentication for host ' + options.hostname);
     options.token = token;
+  } else if (enabled !== undefined) {
```

Comment:
if somebody configured `enabled=true` it will disable
```suggestion
  } else if (enabled === false) {
```